src/wordpress-plugins/yoast-seo.php

```diff
@@ -240,4 +244,14 @@ private function reset_indexables() {
 		\delete_option( 'yoast_migrations_premium' );
 		return \delete_option( 'yoast_migrations_free' );
 	}
+
+	/**
+	 * Resets the indexing notification such that it is shown again.
+	 *
+	 * @param string $reason The indexing reason why the site needs to be reindexed.
+	 */
+	private function reset_indexing_notification( $reason ) {
+		WPSEO_Options::set( 'indexing_reason', $reason );
+		Yoast_Notification_Center::get()->remove_notification_by_id( 'wpseo-reindex' );
```

Comment:
I would use the constant from the `Indexing_Notification_Integration`. In case of a rename we probably might forget this instance.

```suggestion
		Yoast_Notification_Center::get()->remove_notification_by_id( Indexing_Notification_Integration::NOTIFICATION_ID );
```

Reply:
yoast-seo shouldn't have dependencies on integrations; integrations hook into wordpress, integrations may not be needed anywhere else. 
the const should (later) go into a const class as per our decision of tuesday - but we have more consts to move, imho not urgent